### PR TITLE
[Project start] Merge html

### PR DESCRIPTION
**Pull request by vparandii**

### CODE REVIEW

File: src/html/blocks/menu/menu.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<nav class="menu page__menu menu_@@mods">
+    <div class="list action-list">
+        <a href="#" class="list-item list-item_small menu__settings">
+            Settings
+        </a>
+
+        <a href="/task-list.html" class="list-item menu__today">
+            today
+            <div class="badge-group">
+                <i class="badge badge_circle color_yellow"></i>
```
Comment (mirantel) [inline src/html/blocks/menu/menu.html:10]: By design does not need a modifier: "badge_circle". Will be better add CSS properties from "badge_circle" to "circle"

File: src/html/blocks/menu/menu.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                <i class="badge badge_circle color_green"></i>
+            </div>
+        </a>
+
+        <a href="/week-task-list.html" class="list-item menu__next-seven-days">
+            next 7 days
+        </a>
+    </div>
+
+    <div class="separator"> projects</div>
```
Comment (mirantel) [inline src/html/blocks/menu/menu.html:21]: This is header or title of list, so will be better to change class name.

File: src/html/blocks/menu/project-list-item.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<a href="@@href" class="list-item project-item">
+    <span class="project-item__marker color_@@color"></span>
+    <span class="project-item__title"> @@project </span>
+    <div class="project-item__count">
+        <span class="badge badge_circle color_@@color">@@count</span>
+    </div>
+</a>
```
Comment (mirantel) [inline src/html/blocks/menu/project-list-item.html:7]: Will be better use one modifier for changing project colors. project-item_@@color
"project-item marker color@@color"- this block can be created by :before

File: src/html/blocks/task-item.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            @@access_time
+        </span>
+        }
+    </div>
+
+    <div class="task-item__buttons task-item__buttons@@buttons_mod">
+        <button class="button color_green task-item__edit">
+            <i class="icon icon_pencil">
+            </i>
+            <br/>
```
Comment (mirantel) [inline src/html/blocks/task-item.html:42]: BR for creating layout isn't good practice. BR can be used only in user content.

File: src/html/blocks/task-item.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+            edit
+        </button>
+
+        <button class="button color_blue task-item__start">
```
Comment (mirantel) [inline src/html/blocks/task-item.html:46]: Will be better add color styles for this buttons in class "task-item__start" without modifiers "color_blue" or "color_green"

File: src/scss/core/_util.scss [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    margin-right: -.25em; /*might want to tweak this. .25em for extra white space */
+    height: 100%;
+  }
+}
+
+.i-v-box {
+  display: inline-block;
+  vertical-align: middle;
+  white-space: normal;
+}
```
Comment (mirantel) [inline src/scss/core/_util.scss:35]: We haven't block with this class in html.

File: src/scss/core/modifyers/_mixins.scss [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  }
+}
+
+@mixin colors-state-all-for($selector, $text-color: $gray-scale-0) {
+  @each $bg-color-name, $bg-color in $colors {
+    #{$selector}.color_#{$bg-color-name} {
+      @include colorize-all($text-color, $bg-color);
+    }
+  }
+}
```
Comment (mirantel) [inline src/scss/core/modifyers/_mixins.scss:23]: Will be better rename mixins.

File: src/html/blocks/week-calendar.html [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<div class="week-calendar">
```
Comment (mirantel) [inline src/html/blocks/week-calendar.html:1]: I think, will be better use name "week" instead "week-calendar".

File: src/html/blocks/week-calendar-day.html [hunk 9]
```diff
@@ -0,0 +1,5 @@
+<input class="week-calendar__radiobutton" type="radio" name="@@name" id="@@input" @@ischecked />
+<label class="week-calendar-day" for="@@input">
```
Comment (mirantel) [inline src/html/blocks/week-calendar-day.html:2]: I think, will be better use name "week__item" or "week__day" instead "week-calendar-day".

File: src/scss/core/_util.scss [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+  &:before {
+    content: "";
+    display: inline-block;
+    vertical-align: middle;
+    width: 0;
+    margin-right: -.25em; /*might want to tweak this. .25em for extra white space */
+    height: 100%;
+  }
+}
+
```
Comment (mirantel) [inline src/scss/core/_util.scss:30]: We don't use this mixin.

File: src/scss/core/branding/_vars.scss [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+
+$hover-color: rgba(170,180,190, .5);
+
+// gray scale
+$gray-scale-0: #FFF;
+$gray-scale-1: #F1F3F6;
+$gray-scale-2: #D3DBE3;
+$gray-scale-3: #B9C1C9;
+$gray-scale-4: #ABB4BE;
+$gray-scale-5: #57626B;
```
Comment (mirantel) [inline src/scss/core/branding/_vars.scss:26]: I think, colors will be in global _vars.scss

File: src/scss/core/_reset.scss [hunk 12]
```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: $font-family;
+  @include font-regular(14px);
+}
```
Comment (mirantel) [inline src/scss/core/_reset.scss:4]: Will be better add this properties to the src/scss/project/_reset.scss

File: src/html/blocks/menu/menu.html (hunk 1 above)
Comment (vparandii) [inline src/html/blocks/menu/menu.html:10]: https://github.com/frontend-school/Do-While/commit/f653c560b5b05550c9cac1d55ac29aec51c09b27

**Pull request merged**